ci/travis/repository.yml

```diff
@@ -1,12 +1,5 @@
 numpy:
-  1.18.4:
-    source: pypi
-    make: pip
-    maintainer: travis
-    depends:
-      - setuptools
-      - python
-  1.16.6:
+  1.23.5:
```

Comment:
Would you know why we had the two versions before and now we have only one?

Reply:
No, could not find a reason for it in history. But reducing to one version does not break the tests, so then it should be reduced.